roles/kustomize_deploy/tasks/execute_step.yml

```diff
@@ -319,10 +315,7 @@
         - not cifmw_kustomize_deploy_generate_crs_only | bool
       vars:
         hooks: "{{ stage.post_stage_run | default([]) }}"
-        step: "{{ item }}"
+        step: "post_{{ _stage_name_id }}_run"
       ansible.builtin.include_role:
         name: run_hook
       loop:
```

Comment:
Thanks for the patch @hjensas, tiny typo:
```suggestion

```

Reply:
oh, indeed. Good catch! Thanks Lewis! I rebased, and pushed a new version with your suggested change.